src/components/notifications/AppExplorer/AppExplorer.scss

```diff
@@ -85,23 +85,24 @@
 
   &__apps {
     width: 100%;
-    max-width: 700px;
+    max-width: 768px;
```

Comment:
This shouldn't screen width. This container intentionaly set to 700px by the designer. It's not a screen breakpoint.